code/modules/virus2/effect/stage_2.dm

```diff
@@ -260,9 +260,11 @@
 	badness = EFFECT_DANGER_HELPFUL
 	multiplier = 4
 	max_multiplier = 10
+	max_count = 1
 
 /datum/disease2/effect/lantern/activate(var/mob/living/mob)
-	mob.set_light(multiplier)
+	var/color = rgb(rand(0,255), rand(0,255), rand(0,255))
+	mob.set_light(multiplier, l_color = color)
```

Comment:
```suggestion
	mob.set_light(multiplier, multiplier/3,l_color = color)
```
Doing this would increase the mob's light_power as well, making the light source a bit more visible. Right now you often don't realize you've got Lantern Syndrome until you go in a dark place.